languages/python/exercises/concept/lists/lists.py

```diff
@@ -0,0 +1,46 @@
+def to_list(arg1, arg2, arg3, arg4, arg5):
```

Comment:
As with the comments on the `example.py` file, suggest renaming the functions and the arguments to go with the story and/or avoid referencing types and having to use a leading underscore.